genesis/bootstrappers.json

```diff
@@ -0,0 +1,186 @@
+{
+    "1": [
```

Comment:
I wonder if it would be more clear to use the names `mainnet` and `fuji` rather than the numbers

Reply:
Just changed to network name.